packages/pwa-kit-build/src/configs/webpack/config.js

```diff
@@ -225,7 +225,11 @@ const client =
                 entry: {
                     main: './app/main'
                 },
-                plugins: [...config.plugins, new LoadablePlugin({writeToDisk: true})]
+                plugins: [...config.plugins, new LoadablePlugin({writeToDisk: true})],
+                // Hide the performance hints, since we already have a similar `bundlesize` check in `pwa` package
+                performance: {
+                    hints: false
+                }
```

Comment:
I know that hiding the warnings is in the ticket's AC, but Is it the right solution?

I was not able to see the Warnings in V2 running `npm run build`.

Instead of completely hiding the warning, I would suggest investigating why the bundle size increased and if it makes sense the size increase,  update the file sizes here:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/eefc03f4f6a465b2946c839a96e7e39c415047ed/packages/pwa-kit-build/src/configs/webpack/config.js#L209-L212

Reply:
Good question, Adam. I didn't realize too that we set the performance limits in webpack. 

So I took a look at the bundle size increase, and it does make sense to me:
- First of all, with the bundle analyzer, I don't see anything suspicious or significant
- Then I saw in our Slack history that during the hackathon, Oli did not enable the minify option, so the bundle size became huge. But after enabling it, the numbers became much smaller and they looked to be all right with him (even though they're slightly above the limit)

Reply:
I thought to myself, "ok let's update those numbers in the webpack config". But then I became unsure:
- Now we have 2 places where we check the bundle size: in webpack and also in `pwa`'s npm script. It's better to have just a single place to do it.
- Who's the target audience for the file size limit in webpack? It looks to be for ourselves only. Users have no way to update/configure the limit numbers because they're located in a different package (inside `pwa-kit-build`). OR they need to override it from their own webpack config, which may not be straightforward to do.

So IMO my suggestion is to do the following:
- Remove the file size limits from the webpack config
- Still hide Webpack's performance warnings (because by default, it's still checking the bundle size)
- And then update the size limit in pwa's package.json

What do you think? @adamraya

Reply:
You're right @vmarta, it's not easy for users to update the bundle size in the Webpack config, so I agree that is best to hide the warnings there.

One thing I noticed though is that the bundle is still bigger in V2 than V1:



is if run 


`$ npm run test:max-file-size

> scaffold-pwa@2.0.0-dev.2 test:max-file-size /Users/arayanavarro/git/v2-pwa-kit/packages/pwa
> bundlesize


─ build/main.js
  ✖ build/main.js      47.73KB > 40 kB gzip

─ build/vendor.js
  ✔ build/vendor.js    239.94KB < 250 kB gzip

  1 check passed,  1 check failed

Reply:
Yeah, currently we're not communicating to the users about their bundle size. Historically (as far as I can see), the `max-file-size` script was _not_ run together with other npm scripts like `prod:build` etc. I can see how it can be convenient for the users to be told that their bundle size has grown past the limit. On the other hand, it can be a source of friction too for them: since `bundlesize` outputs an error, that may prevent them from pushing a bundle.

Right now I'm thinking to stick with how things used to be. Leave `max-file-size` as a tool that users can choose to use in their workflow, but we're not forcing it on them.

Reply:
Speaking of `max-file-size`: we used to be running it in our CI. But at some point, it's removed for all environments in [this PR](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/92#issuecomment-930497570). But I think we should bring it back such that it runs in non-Windows only.

Reply:
Sounds good @vmarta. Let's leave it as it is. We can always enable warnings later if users start to reach MRT limits too often.

Reply:
Ok, thanks @adamraya . FYI I've made the changes to webpack. Please review whenever you have time, thanks.